### DISCUSSION TITLE
Option to modify weapon enchant glow with transmog feature

### DISCUSSION THREAD

**Discussion by ghost** [chromiecraft/chromiecraft#4800] [Suggestions]
As we all know with transmog feature we are able to modify the looks of our gear, which opened up a lot of optional gameplay in form of farming gear for transmog and improving the quality of overall experience of playing on the server. However there is one feature that is still missing and that is the option to modify the looks of weapon enchantments, both by changing between the different looks and/or at least being able to remove the glow from the weapons if we don't like the look of a particular enchant.

It would also increase the demand of certain low level enchants that have a particularly nice look and might stimulate the economy and allow new players that are leveling enchanting to make some money with crafting said low level enchants for weapons. If we could get a small update so we can do that as well it would be great!

**Comment by 55Honey** [comment]
Hi @Gesche0323,

the transmog-module is not maintained by the ChromieCraft team. Feel free to bring up your suggestion here:
https://github.com/azerothcore/mod-transmog

Kind regards
